### PR TITLE
Adds tickets section

### PR DESCRIPTION
**Pull request by joaojusto**
screen shot 2015-04-13 at 16 10 35

### CODE REVIEW

File: app/_sass/components/_base.sass [hunk 1]
```diff
@@ -10,3 +10,6 @@
 @import separator
 @import code-of-conduct
 @import text
+@import column-container
```
Comment (naps62) [inline app/_sass/components/_base.sass]: like I said previously, this should be renamed to something like ticket-columns. It's not a generic component, the name should indicate so

File: app/index.html [hunk 2]
```diff
@@ -9,3 +9,4 @@
 {% include sections/30_speakers.html %}
 {% include sections/35_speakers-cta.html %}
 {% include sections/60_dont-miss-it-cta.html %}
+{% include sections/40_tickets.html %}
```
Comment (naps62) [inline app/index.html:11]: I guess numbering the files wasn't explicit enough for you :bowtie: 
this should come before 60_dont-miss-it-cta

File: app/_sass/components/_base.sass (hunk 1 above)
Comment (joaojusto) [inline app/_sass/components/_base.sass]: Ok I forgot to rename before making the PR

File: app/index.html (hunk 2 above)
Comment (joaojusto) [inline app/index.html:11]: :smile:

**Comment by naps62** [comment]
:shipit:

**Pull request merged**